dashboard/influx-data-gen/influx-data-gen.lua

```diff
@@ -0,0 +1,77 @@
+-- request-gen.lua
+-- usage: 
+-- lua request-gen.lua
+--
+local socket = require("socket")
+local lfs = require("lfs")
+
+local header = 'https,host=ring-0.io,location=rj'
+local flow = 100 -- ~ request/second  (max: 4000)
+
+math.randomseed(os.time())
+
+function read_file_to_table(file_name)
+    local file = io.open(file_name, "r")
```

Comment:
minor fancy idiomatic suggestion, since we've bumped to lua 5.4:
```suggestion
    local file <close> = io.open(file_name, "r")
```
feel free to ignore.

Reply:
wow. I didn't know that!